src/lib/models/AuthProvider.ts

```diff
@@ -25,6 +25,7 @@ export default class AuthProvider implements AuthData {
   alt: boolean;
   imageSrc?: string | undefined;
   imageAlt?: string | undefined;
+  sessionLogoutURI?: string | undefined;
```

Comment:
What is the difference between
`logouturl?: string | undefined;`
and 
`sessionLogoutURI?: string | undefined;`

Reply:
sessionLogoutURI has been changed to logouturl.